src/bitcoin-cli.cpp

```diff
@@ -642,7 +643,8 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - whether we relay transactions to this peer (relaytxes is false)\n"
+        "           \".\" - we do not relay transactions to this peer (relaytxes not available as peer connection is still being set up)\n"
+        "           \"*\" - we do not relay transactions to this peer (relaytxes is false)\n"
```

Comment:
IMO, it would be worthwhile to use "`.`" to signal we aren't relaying txs (i.e. `relaytxes is false`) and have "`*`" signal the case where the info is unavailable. 

Mainly for two reasons, consistency and intuition. Regarding consistency, `addrp` already uses '`.`' to signal that we aren't relaying addresses (specifically, when `addr_relay_enabled is false`) which is a similar case here, as `relaytxes is false`. Though currently on master, `*` is used to show when `relaytxes is false`, it breaks that consistency also, and I'd say '`*`' intuitively seems more like an undetermined/pending state than '`.`'.

Reply:
It's a slightly larger change of behavior, but sure -- done.